### ISSUE TITLE
[vip-split] [Split Expense] Clean up instances of request terminology in HelpDot

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#40074]
Part of the [Add Split expense to global create](https://github.com/Expensify/App/issues/36748) project, [design doc](https://docs.google.com/document/d/1Z0V68fpdUhx21bsmX-DV0zXx0-w-kRIO00Wpblac9AU/edit).

Please, implement following sections of the design doc:
- [Clean up instances of money request terminology in HelpDot](https://docs.google.com/document/d/1Z0V68fpdUhx21bsmX-DV0zXx0-w-kRIO00Wpblac9AU/edit#bookmark=id.ia8ewo69nird)

This project is handled by our experienced C+: @shubham1206agra and @getusha. Please, refrain from posting proposals

<details><summary>Issue Owner</summary>Current Issue Owner: @laurenreidexpensify</details>

**Comment by laurenreidexpensify** [comment]
https://help.expensify.com/articles/new-expensify/expenses/Request-Money

**Comment by laurenreidexpensify** [comment]
I'm gonna come back to this on Tues / Wed with a plan for updates

**Comment by laurenreidexpensify** [comment]
Okay will try get some momentum on this today and Monday and get this over the line by 3 May

**Comment by laurenreidexpensify** [comment]
~[help.expensify.com/articles/new-expensify/expenses/Distance-Requests](https://help.expensify.com/articles/new-expensify/expenses/Distance-Requests)~ 

Cortney is handling as part of vip-vsb

**Comment by laurenreidexpensify** [comment]
~[help.expensify.com/articles/new-expensify/workspaces/The-Free-Plan](https://help.expensify.com/articles/new-expensify/workspaces/The-Free-Plan)~

False positive, unrelated

**Comment by laurenreidexpensify** [comment]
Okay plot twist, we discovered in review yesterday that Christina was also working on the Request Money page updates [here](https://docs.google.com/document/d/1RhdgMXYwRSUWPed07fW0u17kie9OGrj4qu71kh4F6jg/edit) so we're going to use that doc as the main one for the page updates. 

I've moved the referral page request to its own issue here
- https://github.com/Expensify/App/issues/41346

Next Steps: 
-  the Request Money page updates [here](https://docs.google.com/document/d/1RhdgMXYwRSUWPed07fW0u17kie9OGrj4qu71kh4F6jg/edit)
- Ask Daniel, Kevin and Andrew to review it to to ensure all flows are covered in it accurately 
- Sign off back to Ren for final review and publish

**Comment by laurenreidexpensify** [comment]
Okay plot twist, we discovered in review yesterday that Christina was also working on the Request Money page updates [here](https://docs.google.com/document/d/1RhdgMXYwRSUWPed07fW0u17kie9OGrj4qu71kh4F6jg/edit) so we're going to use that doc as the main one for the page updates. 

I've moved the referral page request to its own issue here
- https://github.com/Expensify/App/issues/41346

Next Steps: 
-  the Request Money page updates [here](https://docs.google.com/document/d/1RhdgMXYwRSUWPed07fW0u17kie9OGrj4qu71kh4F6jg/edit)
- Ask Daniel, Kevin and Andrew to review it to to ensure all flows are covered in it accurately 
- Sign off back to Ren for final review and publish

**Comment by laurenreidexpensify** [comment]
https://expensify.slack.com/archives/C03SC9DVD/p1714499142196349 update

**Comment by laurenreidexpensify** [comment]
- https://github.com/Expensify/App/issues/41346

This still needs to be updated before we can close

**Comment by laurenreidexpensify** [comment]
Still waiting on #41346

**Comment by laurenreidexpensify** [comment]
[expensify.slack.com/archives/C02QSAC6BJ8/p1715110999590789](https://expensify.slack.com/archives/C02QSAC6BJ8/p1715110999590789) - we're doing a deploy freeze for site stability this week, so unlikely we will wrap this up til next week now

**Comment by mountiny** [comment]
@laurenreidexpensify the deploy freeze is up now so feel free to go ahead with the update. Thank you!

**Comment by laurenreidexpensify** [comment]
Ren pointed out this is currently on Expensify Classic site rather than NewDot - I missed that detail, so we need to migrate this over to the correct hub.

**Comment by laurenreidexpensify** [comment]
Waiting for PR to be deployed PR - https://github.com/Expensify/App/pull/42288

**Comment by laurenreidexpensify** [comment]
Still waiting on PR to be deployed - https://github.com/Expensify/App/pull/42288

**Comment by mountiny** [comment]
I believe this is deployed https://help.expensify.com/articles/expensify-classic/expensify-partner-program/Referral-Program

Was this all changes required in the help site? I would have expected a bit more. If yes, we can close this. Thank you Lauren!
